modules/creating-a-data-science-project.adoc

```diff
@@ -25,21 +24,20 @@ ifdef::upstream[]
 endif::[]
 
 .Procedure
-. From the {productname-short} dashboard, click *Data Science Projects*.
-+
-The *Data Science Projects* page opens.
+. From the {productname-short} dashboard, select *Data Science Projects*.
+
 . Click *Create data science project*.
-+
-The *Create a data science project* dialog opens.
-. Enter a *name* for your data science project.
+
+. In the *Create a data science project* dialog, enter a display *name* for your project.
+
 . Optional: Edit the *resource name* for your data science project. The resource name must consist of lowercase alphanumeric characters, '-', and must start and end with an alphanumeric character.
```

Comment:
I know this one is legacy/preexisting, but can also capitalize *resource name* here.